docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3.3'
+services:
+  api:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile-api
+    container_name: api
+    ports:
+      - 8080:8080
+    expose:
+      - 8080
+    environment:
+      NEO4J_URI: bolt://neo4j:neo4j@db:7687
+      DEV: "true"
+    depends_on:
+      db:
+        condition: service_healthy
+  db:
+    image: neo4j:3.4-enterprise
```

Comment:
Will 3.5 work?

Reply:
I think I've checked it and it didn't.  I think I've filed an issue for upgrading neo4j.